lib/project/ext/ui_view.rb

```diff
@@ -82,7 +82,12 @@ def image
   def stylesheet
     rmq.stylesheet
   end
+
   def stylesheet=(value)
     rmq.stylesheet = value
   end
+
+  def find(*args)
+    rmq(self).find(args)
```

Comment:
heh Yeah I figured there was a flatten in there, etc.  anyway I am plan to fix it today :+1: